pkg/controller/istiocontrolplane/inputs.go

```diff
@@ -0,0 +1,91 @@
+package istiocontrolplane
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/helm/pkg/manifest"
+
+	"istio.io/operator/pkg/apis/istio/v1alpha1"
+	"istio.io/operator/pkg/helmreconciler"
+)
+
+// defaultProcessingOrder for the rendered charts
+var defaultProcessingOrder = []string{
+	"istio",
+	"istio/charts/security",
+	"istio/charts/galley",
+	"istio/charts/prometheus",
+	"istio/charts/mixer",
+	"istio/charts/pilot",
+	"istio/charts/gateways",
+	"istio/charts/sidecarInjectorWebhook",
+	"istio/charts/grafana",
+	"istio/charts/tracing",
+	"istio/charts/kiali",
+}
```

Comment:
We should be using istio/installer charts here, these ones are deprecated.

Reply:
at this point, i'd like to leave that as a future task, as what i have working in maistra is based on the charts in istio/istio.